scripts/pre_commit_linter.py

```diff
@@ -666,15 +668,53 @@ def _check_import_order(all_files):
 
     return summary_messages
 
+
+def _check_def_spacing(all_files):
```

Comment:
`def _check_def_spacing(files_to_lint)`

Reply:
All other functions have the same signature to maintain consistency. Should I change this irrespective of that?  
Thanks!

Reply:
Hmm, in that case, leave it as is.